modules/planning/tasks/deciders/path_bounds_decider/path_bounds_decider.cc

```diff
@@ -610,7 +619,8 @@ bool PathBoundsDecider::FindEmergencyPullOverS(
   const double adc_end_s = reference_line_info.AdcSlBoundary().end_s();
 
   // TODO(all): to be implemented
-  *pull_over_s = adc_end_s + 10.0;
+
+  *pull_over_s = adc_end_s + 15.0;
```

Comment:
should this `15` be a config?

Reply:
yes. there's a TODO tag and will be implemented soon